components/list/list-item-mixin.js

```diff
@@ -108,10 +108,13 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				display: none;
 			}
 
-			:host([_tooltip-showing]),
-			:host([_dropdown-open]) {
+			:host([_dropdown-open]),
+			:host([_focusing]) {
 				z-index: 10; /* must be greater than adjacent selected items (if this is increased, d2l-collapsible-panel must be updated too) */
 			}
+			:host([_tooltip-showing]) {
+				z-index: 20; /* must be greater than adjacent selected items (if this is increased, d2l-collapsible-panel must be updated too) */
+			}
```

Comment:
I choose `z-index: 20` here but this value is relative. It could really be anything but it just needs to be `> 10` which is the `_focusing` `z-index` above.

Here is a video of the bug showing what happens if the `_tooltip-showing` index is not higher than 10. You can see that if a lower list item is focused then the previous list item's tooltip will get hidden behind the focused list item.

![tooltip_bug](https://user-images.githubusercontent.com/57574063/226994234-33aff434-7d69-4c5d-995e-173db9681f58.gif)

Reply:
@jesseag I see. The tooltip case makes sense. We don't have the same issue with dropdowns or dialogs opened from the list-item because adjacent items cannot become focused while they are open. I think I would be inclined to tighten the range. If anything comes along that needs to insert itself in the range, I think we/I want to know about it.

@semibran yeah, that's a good suggestion. We've briefly considered introducing some type of `z-index` manager a couple years ago and I generated a list of our usage - it is like the wild wild west. I think a set of CSS variables might help align on, and manage the values, but I think we'd want to take a broader fresh look at `z-index` usage. Probably not a mixin though, since we'd want to be able to use those values in non-component code (i.e. legacy/MVC LMS) as well. I think if we do that, we can do it separate from this PR.